libraries/resource_firewall_rule.rb

```diff
@@ -2,10 +2,11 @@ class Chef
   class Resource::FirewallRule < Resource
     include Poise(Chef::Resource::Firewall)
     provides :firewall_rule
+    attr_accessor :raw
```

Comment:
the parameter is defined there so we could write to this parameter using the zap provider (zap uses this to remove the raw rules which are not defined using chef), i've found alternatives to fix this in the zap provider though.